src/main/java/com/dell/isg/smi/service/device/discovery/controller/DeviceDiscoveryController.java

```diff
@@ -15,7 +15,7 @@
 import org.springframework.web.bind.annotation.RestController;
 
 import com.dell.isg.smi.commons.elm.exception.RuntimeCoreException;
-import com.dell.isg.smi.commons.utilities.CustomRecursiveToStringStyle;
+import com.dell.isg.smi.commons.elm.utilities.CustomRecursiveToStringStyle;
```

Comment:
Change back to utilities.  We moved utilities out of ELM and into their own JAR in the newer elm version.